packages/components/src/Drawer/Drawer.tsx

```diff
@@ -53,16 +59,20 @@ export interface DrawerProps
 // ---------------
 export const Drawer = ({
   children,
-  variant,
-  size,
+  size = 'default',
   open,
   keyboardDismissable,
   closeButton,
   role = 'complementary',
+  placement = 'right',
   ...props
 }: DrawerProps) => {
   const ref = useRef<HTMLElement | null>(null);
-  const classNames = useClassNames({ component: 'Drawer', variant, size });
+  const classNames = useClassNames({
+    component: 'Drawer',
+    variant: placement,
```

Comment:
I think placement should be no variant, what if you have a variant and want to use another position🤔